File: Arrays_Strings/Min_Max.java
package Arrays_Strings;
import java.util.Scanner;
public class Min_Max {

    static int min(int arr[]){
        int min = arr[0];
        for(int num: arr){
            if(min>num){
                min = num;
            }
        }
        System.out.println(min);
        return min;
    }

    static int max(int arr[]){
        int max = arr[0];
        for(int num: arr){
            if(max<num){
                max = num;
            }
        }
        System.out.println(max);
        return max;
    }

    public static void main(String[] args){
        
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int[] arr = new int[n];
        for(int i=0; i<n; i++){
            arr[i] = input.nextInt();
        }
        int maxno = max(arr);
        int minno = min(arr);
        System.out.println(maxno+minno);
    }
}

File: Arrays_Strings/Min_Max.py
def findSum(A, N): 
    max=A[0]
    min=A[0]
    for i in range(N):
        if max<A[i]:    max=A[i] 
    for i in range(N):
        if min>A[i]:   min=A[i]
    return max+min

N =5
A = [4, 3, 8, 1, 2]
print(findSum(A, N))
